contracts/seed/Seed.sol

```diff
@@ -375,7 +373,7 @@ contract Seed {
         Lock storage tokenLock = tokenLocks[_locker];
 
         // Check cliff was reached
-        uint elapsedTime = _currentTime().sub(tokenLock.startTime);
+        uint elapsedTime = _currentTime().sub(startTime);
```

Comment:
I have a doubt here. I mean, when we call this function, it should calculate daysVested for each lock. But each locked might have been vested on different days. Due to this, the daysVested should be different for different locks. 
So here, we are calculating using global var startTime, which will result in same daysVested for all locks.

PS- According to what I understood about this contract.